File: public/usage-examples/geometry/random_bitmap_point/random_bitmap_point-1-simple.py
from splashkit import *

# Create Window
open_window("random bitmap point", 600, 600)
bmp = create_bitmap("random quads",600,600)

#  create quad using random points on bitmap
q = quad_from_points(
    random_bitmap_point(bmp),
    random_bitmap_point(bmp),
    random_bitmap_point(bmp),
    random_bitmap_point(bmp))
draw_quad_on_bitmap(bmp, color_black(), q)

clear_screen(color_white_smoke())

# Draw the bitmap
draw_bitmap(bmp, 0,0)

refresh_screen()
delay(5000)
close_all_windows()

File: public/usage-examples/geometry/random_bitmap_point/random_bitmap_point-1-simple.cpp
#include "splashkit.h"

int main()
{
    // Create Window
    open_window("Random Quad Shape", 600, 600);
    bitmap bmp = create_bitmap("Random Quads", 600, 600);

    // Create quad using random points on bitmap
    quad q = quad_from(
        random_bitmap_point(bmp),
        random_bitmap_point(bmp),
        random_bitmap_point(bmp),
        random_bitmap_point(bmp));
    draw_quad_on_bitmap(bmp, color_black(), q);

    clear_screen(color_white_smoke());

    // Draw the bitmap
    draw_bitmap(bmp, 0, 0);

    refresh_screen();
    delay(5000);
    close_all_windows();

    return 0;
}
